examples/deploy.rs

```diff
@@ -0,0 +1,65 @@
+use anyhow::{anyhow, bail};
+use revm::{
+    primitives::{bytes::Bytes, hex, EthereumWiring, ExecutionResult, Output, TxKind, U256},
+    Evm, InMemoryDB,
+};
+
+/// [ PUSH1, 0x01, PUSH1, 0x17, PUSH1, 0x1f, CODECOPY, PUSH0, MLOAD, PUSH0, SSTORE ]
+const INIT_CODE: &[u8] = &[
+    0x60, 0x01, 0x60, 0x17, 0x60, 0x1f, 0x39, 0x5f, 0x51, 0x5f, 0x55,
```

Comment:
Nit, you could use exact notion from the comment if you include, `revm::interpreter::opcode::*`